### PR TITLE
surname removal

### PR DESCRIPTION
**Pull request by GursharanSingh123**

### CODE REVIEW

File: gg.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
-gursharan singh'
+gursharan 
 no
+yes
```
Comment (toobie-2b) [inline gg.txt:3]: why you typed yes?

File: gg.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
-gursharan singh'
+gursharan
```
Comment (toobie-2b) [inline gg.txt:1]: why was this change required?

Comment (GursharanSingh123) [reply gg.txt:1]: aise hi
